snippets/fsh-snippets.json

```diff
@@ -16,21 +16,47 @@
 		"prefix": ["ext", "Ext"],
 		"body": [
 			"Extension: $1",
-			"Id: ${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}",
-			"Title: \"${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}\"",
-			"Description: \"$2\"",
+			"Id: ${2:${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}}",
+			"Title: \"${3:${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}}\"",
+			"Description: \"$4\"",
 			"* $0"
 		],
 		"description": "Create a FSH Extension"
 	},
 
+	"FSH Logical": {
+		"prefix": ["log", "Log"],
+		"body": [
+			"Logical: $1",
+			"Parent: ${2:Base}",
+			"Id: ${3:$1}",
```

Comment:
Is the `Id` for Logicals typically the same as the name, or is it usually kebab cased (I think that's what it's called), similar to how the Profile and Extension snippet works?

Reply:
Yeah, for logicals and resources it seems the id is usually the same as the name (for whatever reason).  It's a little less clear for logicals but more clear for resources.  For example, the id for `MedicationRequest` is `MedicationRequest`, not `medication-request`.  In many cases, logicals and resources are similar, so I'm sort of leaning into that here.